core/modules/cms/store/block/mutations.ts

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue'
+import { MutationTree } from 'vuex'
+import { entityKeyName } from '@vue-storefront/store/lib/entities'
+import * as types from './mutation-types'
+import CmsBlockState from '../../types/CmsBlockState'
+
+const mutations: MutationTree<CmsBlockState> = {
+  /**
+   * Store CMS Blocks by identifier in state and localForage
+   * @param {} state
+   * @param {Array} cmsBlocks
+   */
+  [types.CMS_BLOCK_UPDATE_CMS_BLOCKS] (state, cmsBlocks) {
+    // save cmsBlocksList data to the state and DB
+    state.cmsBlocks = cmsBlocks
+
+    console.log('cmsBlocks mutation', cmsBlocks.length)
+
+    for (let cmsBlock of cmsBlocks) {
+
+      const cmsBlocksCollection = Vue.prototype.$db.cmsBlocksCollection
```

Comment:
like in `store/block/actions.ts` use initCacheStorage please. Vue instance shouldn't be a place to access data. Only vuex store should be responsible for accessing data and it can take this data from cache/server